### ISSUE TITLE
sorry to tell you the code has been detected by VAC and Value

### ISSUE THREAD

**Issue by KGB121** [Fr0go1/Aeonix-Cs2#28]

**Comment by Corgansm** [comment]
I dont believe this for a second, I've been using Tkazer's sig scanner which is the cheat derived from this project and vac has not detected my cheat yet. I do a ton of testing every day to see if it's detected before I use it on my current prime account.

**Comment by KGB121** [comment]
believe it or not，but it is the fact that i had been banned 3accounts last night

**Comment by Corgansm** [comment]
> believe it or not，but it is the fact that i had been banned 3accounts last night  

Are you using any features that write to memory like trigger bot or aimbot? Features that write to memory are things that do things in game that you didn't do manually like firing or aiming.

**Comment by Fr0go1** [comment]
> > believe it or not，but it is the fact that i had been banned 3accounts last night
> 
> Are you using any features that write to memory like trigger bot or aimbot? Features that write to memory are things that do things in game that you didn't do manually like firing or aiming.

Triggerbot doesnt write

**Comment by Corgansm** [comment]
> > > believe it or not，but it is the fact that i had been banned 3accounts last night
> 
> > 
> 
> > Are you using any features that write to memory like trigger bot or aimbot? Features that write to memory are things that do things in game that you didn't do manually like firing or aiming.
> 
> 
> 
> Triggerbot doesnt write

I just checked you're right, it was pissin me off bc I'd be testing stuff then I'd be looking at someone and try to alt tan and it would not let me. Add mouse_event to aimbot ( add a check for if you're on the cs2 screen )

**Comment by Fr0go1** [comment]
> _No description provided._

https://github.com/Fr0go1/Aeonix-Cs2/discussions/5

**Comment by Fr0go1** [comment]
Click the link

**Comment by anti2rich** [comment]
Def hasn't been detected lol been using the code from this for nearly 3 days straight now dropping 30 bombs every game and still not banned or even bad trust lol, just a skill issue.

**Comment by DanTheEpicMan** [comment]
> Def hasn't been detected lol been using the code from this for nearly 3 days straight now dropping 30 bombs every game and still not banned or even bad trust lol, just a skill issue.

insane how a person can do so little research and be so confident. It's a free user mode cheat, of course its detected, you don't think that VAC employees can use GitHub to look for free cheats and enter them into their database. The only thing this is useful for in for reference for a private cheat.

https://en.wikipedia.org/wiki/Valve_Anti-Cheat
